org.jacoco.core.test/src/org/jacoco/core/test/TargetLoader.java

```diff
@@ -52,17 +52,24 @@ public TargetLoader() {
 	}
 
```

Comment:
@marchof  method `TargetLoader.add(Class<?> source)` is unused. Moreover seems that after this change there is no need in map `classes` in `TargetLoader` since it will be used only to load a single class, what is in fact stated in his javadoc already :wink: